cmd/awslambda/function/main.go

```diff
@@ -5,16 +5,23 @@ package main
 
 import (
 	"context"
+	"fmt"
 	"os"
 
 	"github.com/aws/aws-lambda-go/lambda"
 	"go.uber.org/zap"
 
 	"github.com/cerbos/cerbos/internal/observability/logging"
 	"github.com/cerbos/cerbos/internal/server/awslambda"
+	"github.com/cerbos/cerbos/internal/util"
 )
 
 func main() {
+	if len(os.Args) > 1 && (os.Args[1] == "--version" || os.Args[1] == "-v") {
+		fmt.Printf("cerbos %s\n", util.Version) //nolint:forbidigo
+		return
+	}
```

Comment:
Isn't this only used for the just recipe below? Otherwise, even though it's only a single argument, it's better to use Kong for consistency. It'd also handle unknown arguments properly.

In practice, we should always be doing the release from CI so I'd make the `VERSION` parameter mandatory for the just recipe and fail if the version is not provided.